### ISSUE TITLE
[Feature Request] Double Click / Enter Expected behaviour

### ISSUE THREAD

**Issue by pantar85** [gpodder/gpodder#228]
**Current Behaviour**

Double Clicking or pressing Enter on a selected episode opens up the Episode Details segment.

**Expected Behaviour**

Double Clicking or pressing Enter on a selected episode should try and play the downloaded file, otherwise try to stream it.

**Comment by southvinement** [comment]
Good idea.

**Comment by funkypitt** [comment]
It's an excellent idea, and an absolutely necessary one if one is using gpodder on a tablet pc without keyboard, as I am. What surprises me is that it was already integrated as a setting a long time ago:

https://bugs.gpodder.org/show_bug.cgi?id=38

It seems to have been removed since, or is there something that I missed?

**Comment by pantar85** [comment]
Can't find or figure out how to use this hidden option. If it has been removed does anyone know why?

The default option feels so unnatural. Imagine opening a jpeg, only to be shown it's dimensions, ownership and size... this feels like that. It seems like a mistake but if it's not can anyone run me through it. Cheers.

**Comment by elelay** [comment]
> Imagine opening a jpeg, only to be shown it's dimensions, ownership and size

Imagine you double-click an email subject in your mail client and it opens attachments instead of showing the email contents ;-)
OK this example it's slightly over the board, but I do always first look at show notes before playing an episode and I sincerely think current behaviour is the best one.
However, feel free to provide a pull request with a hidden option to play or stream on double click.
@funkypitt you don't need a keyboard to play an episode : just right-click and select play or left-click and use the play button in the toolbar.

**Comment by funkypitt** [comment]
@elelay "just" right-click and select play? :-)

Playing audio is one of the main functions of any podcast app, and having to navigate to contextual menus, or anything that requires 2 steps to do it is weird to me. Gpodder is pretty much alone is that design choice; most podcast apps that I used on android or desktop linux make playing the audio the default pressing / double-clicking action. 

I'm not asking for a change in the default behavior: to each his own taste, and I fully respect the current developers' choices, but adding (or rather re-enabling, as it used to be present) the option to configure what enter / double-clicking does would be great for users like @pantar85 and myself, and would not change anything for all other users.

BTW I have issue with right-click on my tablet as well, so I ended up using Cpod (cumulonimbus) on my x86 tablet, while I keep gpodder on my desktop, and that works well for me. But I'm used to, and prefer Gpodder and it would be great to have it on all my devices again.

**Comment by thunktone** [comment]
If this hadn't been suggested I was going to suggest it myself. It's really weird to me to make a layout change easier than the main purpose of the app. I would probably go a step further than others and suggest that this should be the default behaviour with an option to change it for those that like to show and hide the description panel frequently.

**Comment by mmortal03** [comment]
I keep double-clicking by force of habit and it opens the show notes instead of playing. This has got to be made an option again.  :)

**Comment by amirrammari** [comment]
+1

**Comment by comodoro** [comment]
This seems to have fizzled out somehow. As a new user, the GUI choices surprise me. There is Shift+Enter for this, for some reason not Enter as one would expect. Furthermore, I cannot play a podcast before it has been downloaded.

**Comment by auouymous** [comment]
As mentioned in #1256, it would be nice to have named actions that could be typed in using :, or bound to keys, key combos, and double-click. The user is then free to bind any available action to double-click instead of a limited set we selected for them. There is also double-clicking a channel which currently opens its dialog, but could be configured to update it. And extensions could also register actions, so double-click could be `Download with Youtube-DL`.

**Comment by elelay** [comment]
> it would be nice to have named actions

yes, seems an interesting idea to explore. Amazing how a simple application such as a podcatcher expands in a fractal of features...
